unified_targets/configs/TCMMF405BLE.config

```diff
@@ -0,0 +1,232 @@
+# Betaflight / TCMMF405BLE (MKF4) 4.0.0 Apr 24 2019 / 09:25:23 (22b9f3453) MSP API: 1.41
+
+# start the command batch
+batch start
+
+board_name TCMMF405BLE
+manufacturer_id TCMM
+
+# resources
+resource BEEPER 1 C13
+resource MOTOR 1 C06
+resource MOTOR 2 C07
+resource MOTOR 3 C08
+resource MOTOR 4 C09
+resource MOTOR 5 A15
+resource MOTOR 6 A08
+resource MOTOR 7 B08
+resource MOTOR 8 NONE
```

Comment:
The lines containing NONE and the settings below that set defaults should not be in here - they make the target more likely to break on future firmware changes.
To fix this, apply the changes to a Unified Target firmware, and then supply the output of diff bare as the Unified Target configuration.